src/main/resources/assets/extraalchemy/lang/it_it.json

```diff
@@ -154,6 +154,14 @@
 	"extraalchemy.config.general.enable_brewing_stand_fire.tooltip1": "Abilitare questa opzione fa si che gli alambicchi vengano", 
 	"extraalchemy.config.general.enable_brewing_stand_fire.tooltip2": "scaldati ogni 2 secondi se c'è una fonte di calore sotto.",
 	
+	"extraalchemy.config.general.brewing_stand_heat_increment_delay": "Brewing stand heat accumulation speed",
```

Comment:
Just remove these lines completely, they will automatically fall back to English if missing